pubsub/gossipsub/gossipsub-v1.2.md

```diff
@@ -56,6 +56,7 @@ This section lists the configuration parameters that needs to agreed on across c
 When the peer receives the first message instance it immediately broadcasts 
 (not queue for later piggybacking) `IDONTWANT` with the `messageId` to all its mesh peers. 
 This could be performed prior to the message validation to further increase the effectiveness of the approach.    
+Additionally, on networks where multiple messages share the same messageId, a user may choose to broadcast IDONTWANT when publishing.
```

Comment:
I think it's good to note that the publisher still shouldn't publish the message to the peers that sent IDONTWANT to the publisher before.

It's trivial to know this but I'm afraid that some implementers will forget this.